csrc/kernel_cache.cpp

```diff
@@ -1242,6 +1242,19 @@ std::vector<at::Tensor> FusionKernelRuntime::runKernelWithInput(
   return outputs;
 }
 
+std::vector<std::unique_ptr<Fusion>> FusionKernelRuntime::getFusionSegments() {
```

Comment:
Out of curiosity, is the natural order in fusion segments not stable enough for your validation?

i.e. `this->fusionSegments()->groups()`.

A nitpick, but if this is only used in test validation, I'm leaning towards not adding an API in FusionKernelRuntime.